src/rust/grapl-observe/src/metric_reporter.rs

```diff
@@ -110,11 +119,35 @@ where
     example: the time to complete rendering of a web page for a user.
     */
     pub fn histogram(
+        &mut self,
+        metric_name: &str,
+        value_millis: f64,
+        tags: &[TagPair],
+    ) -> Result<(), MetricError> {
+        self.write_metric(metric_name, value_millis, MetricType::Histogram, None, tags)
+    }
+
+    /**
+     * In order to shoehorn units into the statsd protocol, we specify a
+     * special "_unit" tag that will
+     * be popped off in the metric forwarder.
+     */
+    pub fn histogram_with_units(
         &mut self,
         metric_name: &str,
         value: f64,
+        unit: HistogramUnit,
         tags: &[TagPair],
```

Comment:
If we're going to be converting to a Vec anyways we should just take a Vec, or something like:
```
impl Into<Vec<TagPair>>
```
And then instead of `.to_vec` we'd call `.into()`